diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c77a770..3d9241b7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,7 +47,7 @@ set_package_properties(FLEX PROPERTIES
     PURPOSE "Required for the Predicate parser"
 )
 
-find_package(BISON 3.0 REQUIRED)
+find_package(BISON REQUIRED)
 set_package_properties(BISON PROPERTIES
     URL "http://www.gnu.org/software/bison"
     DESCRIPTION "general-purpose parser generator"
