diff --git a/cmake/FindHUNSPELL.cmake b/cmake/FindHUNSPELL.cmake
index 1765cec..6abf96d 100644
--- a/cmake/FindHUNSPELL.cmake
+++ b/cmake/FindHUNSPELL.cmake
@@ -17,7 +17,7 @@ find_path(HUNSPELL_INCLUDE_DIRS
           HINTS ${PKG_HUNSPELL_INCLUDE_DIRS}
 )
 find_library(HUNSPELL_LIBRARIES
-             NAMES ${PKG_HUNSPELL_LIBRARIES} hunspell hunspell-1.6 hunspell-1.5 hunspell-1.4 hunspell-1.3 hunspell-1.2 libhunspell
+             NAMES ${PKG_HUNSPELL_LIBRARIES} hunspell hunspell-1.7 hunspell-1.6 hunspell-1.5 hunspell-1.4 hunspell-1.3 hunspell-1.2 libhunspell
              HINTS ${PKG_HUNSPELL_LIBRARY_DIRS}
 )
 
diff --git a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
index 575f7ef..daa8893 100644
--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -38,7 +38,7 @@ target_sources(KF6SonnetCore PRIVATE
 # create trigrams file + add trigrams resource
 if (TARGET KF6::parsetrigrams)
     add_custom_command(OUTPUT "${PROJECT_BINARY_DIR}/data/trigrams.map"
-                       COMMAND KF6::parsetrigrams "${PROJECT_SOURCE_DIR}/data/trigrams" > "${PROJECT_BINARY_DIR}/data/trigrams.map")
+                       COMMAND ${CMAKE_COMMAND} -E env "PATH=${CMAKE_INSTALL_PREFIX}/bin" KF6::parsetrigrams "${PROJECT_SOURCE_DIR}/data/trigrams" > "${PROJECT_BINARY_DIR}/data/trigrams.map")
     configure_file(${PROJECT_SOURCE_DIR}/data/trigrams.qrc.in "${PROJECT_BINARY_DIR}/data/trigrams.qrc" @ONLY)
     target_sources(KF6SonnetCore PRIVATE "${PROJECT_BINARY_DIR}/data/trigrams.qrc")
 endif()
